### DISCUSSION TITLE
Classification model for fMRI

### DISCUSSION THREAD

**Discussion by EAWakelin** [Project-MONAI/MONAI#6933] [Q&A]
Hi,

I am pretty new to deep learning and MONAI and am interested in extending the 3D image classification tutorial to 4D images like functional MRI.

Does MONAI currently allow this type of imaging and what would be the best way to go about building this model?

Thanks!

**Comment by KumoLiu** [comment]
Hi @EAWakelin, thanks for your interest here and welcome to your contribution.

You can refer to the contribution guide doc:
https://github.com/Project-MONAI/MONAI/blob/dev/CONTRIBUTING.md

Thanks!

**Comment by EAWakelin** [comment]
Thanks Kumo,

OK, just so I'm clear, that functionality does not yet exist?

If so, I'll have to start thinking!
